src/Router.php

```diff
@@ -35,20 +35,24 @@ class Router
     private $globalRoutePrefix;
     /** @var string $cacheDirectory */
     private $cacheDirectory;
+    /** @var string $controllersSuffix */
+    private $controllersSuffix;
 
     /**
      * Router constructor.
      * @param App $app
      * @param string|string[] $controllersDirectory
      * @param string $globalRoutePrefix used to give a global URI prefix
+     * @param string $controllersSuffix
      * @param string|null $cacheDirectory if null, cache is disabled
      */
-    public function __construct(App $app, $controllersDirectory, $globalRoutePrefix = '', $cacheDirectory = null)
+    public function __construct(App $app, $controllersDirectory, $globalRoutePrefix = '', $controllersSuffix = 'Controller', $cacheDirectory = null)
```

Comment:
This is a breaking change, which is not reported in the README.md file.

Also the README.md file should be updated reguarding the need (from MUST to DEFAULT) to have 'Controller' as the class' suffix.